### PR TITLE
test(werewolves-assistant-web): add ci build and tests

### PR DESCRIPTION
**Pull request by antoinezanardi**
## 🚀 Why this PR

As suggested by @manniL, I added the CI steps of building and testing my Werewolves Assistant Nuxt app. It's a quick PR, I probably missed some things, feel free to let me know if it's the case.

You can check the base repository **[here](https://github.com/antoinezanardi/werewolves-assistant-web-next)**.

Thanks again for your interest for my project ❤️

### CODE REVIEW

File: tests/werewolves-assistant.ts [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import { runInRepo } from '../utils.ts'
+import type { RunOptions } from '../types.ts'
+
+export async function test(options: RunOptions) {
+	await runInRepo({
+		...options,
+		repo: 'antoinezanardi/werewolves-assistant-web',
+		build: ['build'],
+		test: [
+			'lint:fix',
```
Comment (manniL) [inline tests/werewolves-assistant.ts:10]: Do you think the lint command would help finding regressions for a new nuxt version?
If not, we could remove it here ☺️

Comment (antoinezanardi) [reply tests/werewolves-assistant.ts]: You're right, it was not relevant, I removed it 😉

**Review by manniL — APPROVED** [review]

**Comment by danielroe** [comment]
This currently doesn't run as it expects Node >= 21.

**Comment by antoinezanardi** [comment]
> This currently doesn't run as it expects Node >= 21.
> 
> https://github.com/antoinezanardi/werewolves-assistant-web-next/blob/main/package.json#L138-L140
> 
> Any chance you could change that upstream @antoinezanardi?

I can make a quick PR for that, it's not mandatory to use Node 21, but 20 at least. 👍

**Comment by antoinezanardi** [comment]
[Associated PR](https://github.com/antoinezanardi/werewolves-assistant-web-next/pull/309)

**Review by danielroe — APPROVED** [review]

**Pull request merged**